App/Views/SearchView.swift

```diff
@@ -0,0 +1,1238 @@
+        Text(result.postedDateTime)
+            .font(.footnote)
+            .foregroundColor(theme[color: "listSecondaryTextColor"])
+    }
+    
+    private var resultBlurb: some View {
+        let blurbWithMarkdown = result.blurb
+            .replacingOccurrences(of: "<em>", with: "**", options: .caseInsensitive)
+            .replacingOccurrences(of: "</em>", with: "**", options: .caseInsensitive)
+            .trimmingCharacters(in: .whitespacesAndNewlines)
+        
+        let text: Text
+        if var attributedString = try? AttributedString(markdown: blurbWithMarkdown) {
+            attributedString.foregroundColor = theme[color: "listTextColor"]
+            for run in attributedString.runs where run.inlinePresentationIntent == .stronglyEmphasized {
+                if let tintColor = theme[color: "tintColor"] {
+                    attributedString[run.range].foregroundColor = tintColor
+                }
+            }
+            text = Text(attributedString)
+        } else {
+            text = Text(blurbWithMarkdown)
+                .foregroundColor(theme[color: "listTextColor"])
+        }
+        
+        return text
+            .font(.body)
+            .fixedSize(horizontal: false, vertical: true)
+    }
+}
+
+// MARK: - Preview Provider
+struct SearchResultCard_Previews: PreviewProvider {
+    static let testTheme = Theme.theme(named: "brightLight") ?? Theme.defaultTheme()
+    
+    static var previews: some View {
+        Group {
+            VStack(spacing: 16) {
+                SearchResultCard(result: SearchResult(
+                    threadTitle: "Thread title blah blah blah Thread title blah blah blah",
+                    resultNumber: "1.",
+                    blurb: "This is a test blurb that shows how the card handles multiple lines of text in a more natural way",
+                    forumTitle: "Test Forum",
+                    postID: "123",
+                    userName: "TestUser",
+                    postedDateTime: "by Someone in ForumA at Jul 1, 2023 8:04 PM"
+                ))
+                
+                SearchResultCard(result: SearchResult(
+                    threadTitle: "Short title",
+                    resultNumber: "2.",
+                    blurb: "Short blurb",
+                    forumTitle: "Another Forum",
+                    postID: "456",
+                    userName: "AnotherUser",
+                    postedDateTime: "by Someone in ForumB at Jul 2, 2023 9:04 PM"
+                ))
+            }
+            .padding()
+            .previewLayout(.sizeThatFits)
+        }
+        .environment(\.theme, testTheme)
+    }
+}
+
+struct SearchView: View {
+    @StateObject private var model: SearchPageViewModel
+    @SwiftUI.Environment(\.theme) var theme
+    @SwiftUI.Environment(\.dismiss) var dismiss
+    @FocusState private var isSearchFieldFocused: Bool
+    
+    init() {
+        _model = StateObject(wrappedValue: SearchPageViewModel())
```

Comment:
let's pass in the view model instead of creating it here. that'd make it an `@ObservedObject`. we can keep the instance alive by subclassing UIHostingController and adding a property to hold it.